dts/bindings/misc/espressif,esp32-rmt.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2025 Joel Guittet (joelguittet@gmail.com)
+# SPDX-License-Identifier: Apache-2.0
+
+description: ESP32 RMT
```

Comment:
Add a title, spell out the acronym according to the guidelines, and consider adding an actual description :) (bonus point for including an example of a dts snippet as well)